src/adapter/http/src/general/dataset_info_handler.rs

```diff
@@ -29,18 +29,25 @@ use crate::axum_utils::ensure_authenticated_account;
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-#[derive(Debug, serde::Serialize)]
+#[derive(Debug, serde::Serialize, utoipa::ToSchema)]
 #[serde(rename_all = "camelCase")]
 pub struct DatasetOwnerInfo {
+    #[schema(value_type = String)]
     pub account_name: AccountName,
+
+    #[schema(value_type = Option<String>)]
     pub account_id: Option<AccountID>,
 }
 
-#[derive(Debug, serde::Serialize)]
+#[derive(Debug, serde::Serialize, utoipa::ToSchema)]
 #[serde(rename_all = "camelCase")]
 pub struct DatasetInfoResponse {
+    #[schema(value_type = String)]
     pub id: DatasetID,
+
     pub owner: Option<DatasetOwnerInfo>,
+
+    #[schema(value_type = String)]
```

Comment:
Maybe: should we think of a stricter schema format for typical ODF data types?
I.e., length & regex-based restrictions.

Reply:
Yep, we should definitely do it, but would like to leave it out of this PR. I'd like to unblock the work of embedding OpenAPI docs in https://docs.kamu.dev while we polish the documentation in code.